src/Symfony/Component/Notifier/Bridge/Mobyt/MobytTransport.php

```diff
@@ -33,11 +33,15 @@ final class MobytTransport extends AbstractTransport
     private $from;
     private $typeQuality;
 
-    public function __construct(string $accountSid, string $authToken, string $from, string $typeQuality, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(string $accountSid, string $authToken, string $from, string $typeQuality = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
This BC break should not bother anyone but its more the way, how other transports deal with optional parameters 👍

Reply:
This signature change is **not** a BC break.

Reply:
Indeed you are right 👍  thanks

Reply:
Updated